apps/docs/src/components/navigation/navigation.module.css

```diff
@@ -0,0 +1,73 @@
+.Navigation {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  flex-grow: 1;
+}
+
+.Separator {
+  color: var(--borderColor-default);
+  font-weight: var(--base-text-weight-light);
+}
+
+.HorizontalList {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  flex-grow: 1;
+  margin: 0;
+  padding: 0;
+  gap: var(--base-size-2);
+  padding-right: var(--base-size-16);
+  justify-content: flex-end;
+}
+
+.Link {
+  font-family: var(--brand-fontStack-sansSerif);
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  color: var(--fgColor-muted);
+  text-decoration: none;
+  padding: 0 var(--base-size-12);
+  height: var(--base-size-32);
+  border: 1px solid transparent;
+  border-radius: var(--borderRadius-medium);
+}
+
+.DropdownLink {
+  font-family: var(--brand-fontStack-sansSerif);
+  color: var(--fgColor-muted);
```

Comment:
FYI this color token isn't loaded on the page so looks like this 👇 

![Screenshot 2024-12-03 at 09 59 09](https://github.com/user-attachments/assets/acb19134-29ea-4e1f-a7d3-ad47aa272930)

Reply:
Suggestion to fix color and size to match new docs:

```suggestion
  color: var(--brand-color-text-muted);
  font-size: var(--brand-text-size-400);
```

Reply:
Very strange - I thought I already fixed the CSS vars in another commit. Let me make sure I didn't stash it somewhere by accident...

Reply:
Can't find my changes, but I can take yours. Thank you!!